.changeset/great-queens-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
I think this should be a patch since it's causing an unexpected behaviour before. We usually reserve minors for new features.

Reply:
```suggestion
'astro': patch
```